src/main/java/com/elytradev/concrete/inventory/gui/client/GuiHelper.java

```diff
@@ -0,0 +1,233 @@
+/*
```

Comment:
Not happy with this rename - it implies that the class does something other than drawing, which isn't true. In fact, I want to avoid "helper" whenever possible, it doesn't say *anything* about what the class does.